src/Fixer/PhpTag/BlankLineAfterOpeningTagFixer.php

```diff
@@ -86,6 +81,7 @@ public function getPriority()
      */
     public function isCandidate(Tokens $tokens)
     {
-        return $tokens->isTokenKindFound(T_OPEN_TAG);
+        // ignore files with short open tag and ignore non-monolithic files
+        return $tokens[0]->isGivenKind(T_OPEN_TAG) && $tokens->isMonolithicPhp();
```

Comment:
yes, however I noticed a bunch of fixers rely on the candidate check, same for example that a lot of fixers won't run correctly on an empty token collection while this is valid (i.e. empty file)